packages/visual-dev/src/components/Dropdown/Dropdown.tsx

```diff
@@ -160,10 +161,17 @@ const DropdownContentDiv = styled("div")<
   ${Styles.ContentDiv}
   border-radius: ${(props) => props.borderRadius};
 
+  ${(props) => props.popUpwards && `top: 2px; transform: translateY(-100%);`}
+
+  ${(props) =>
+    !props.pill &&
+    (props.popUpwards ? `border-bottom: none;` : `border-top: none;`)}
+
   ${(props) =>
-    props.popUpwards
-      ? `top: 2px; transform: translateY(-100%); border-bottom: none;`
-      : `border-top: none;`}
+    props.pill &&
+    (props.popUpwards
+      ? "margin-bottom: var(--sq-spacing-x-small)"
```

Comment:
`margin-bottom` on `popUpwards` doesn't seem to work with translating. If popUpwards also needs the spacing, will need to translate upwards more with `transform: translateY(calc(-100% - var(--sq-spacing-x-small)))`

Reply:
Issue resolved with the above transform code.